packages/@aws-cdk/aws-elasticloadbalancingv2-actions/README.md

```diff
@@ -0,0 +1,21 @@
+# Actions for AWS Elastic Load Balancing V2
+<!--BEGIN STABILITY BANNER-->
+---
+
+![cdk-constructs: Experimental](https://img.shields.io/badge/cdk--constructs-experimental-important.svg?style=for-the-badge)
+
+> The APIs of higher level constructs in this module are experimental and under active development. They are subject to non-backward compatible changes or removal in any future version. These are not subject to the [Semantic Versioning](https://semver.org/) model and breaking changes will be announced in the release notes. This means that while you may use them, you may need to update your source code when upgrading to a newer version of this package.
+
+---
+<!--END STABILITY BANNER-->
+
+This package contains integration actions for ELBv2. See the README of the `@aws-cdk/aws-elasticloadbalancingv2` library.
+
+## Cognito
+
+To use a Cognito UserPool for authentication, use `AuthenticateCognitoAction`. For details on the
```

Comment:
I like to start off with giving a sentence or two on what this is. Something like -

"ELB allows for requests to be authenticated against a Cognito user pool using the `AuthenticateCognitoAction`......"